### DISCUSSION TITLE
Support for Bitcoin

### DISCUSSION THREAD

**Discussion by rfreis** [WalletConnect/walletconnect-monorepo#805] [Developer Support]
Hi guys!

I would like to use walletconnect to request general payments to my wallet and this is working perfectly for Ethereum and tokens but the examples I've seen so far don't support Bitcoin.

Do you have plans to support Bitcoin ?

Thanks and congratulations for building this project!

**Comment by stpoa** [comment]
Dear WalletConnect Team,

I hope you're doing great! I'm a big fan of your project and have been using it for Ethereum and tokens payments. I noticed that Bitcoin is not currently supported, and I was wondering if there are any plans to add it in the future?

Thank you for all your hard work and contributions to the crypto community.

Best regards
